WalletWasabi.Fluent/Views/Wallets/WalletSettingsView.axaml

```diff
@@ -13,16 +13,27 @@
                  EnableNext="True" NextContent="Done"
                  EnableCancel="{Binding EnableCancel}"
                  EnableBack="{Binding EnableBack}">
-    <StackPanel Classes="settingsLayout">
-      <DockPanel IsVisible="{Binding IsHardwareWallet}">
-        <TextBlock Text="PSBT workflow" />
-        <ToggleSwitch IsChecked="{Binding PreferPsbtWorkflow}" />
-      </DockPanel>
+    <StackPanel Spacing="20">
+      <StackPanel Classes="settingsLayout">
+        <DockPanel IsVisible="{Binding IsHardwareWallet}">
+          <TextBlock Text="PSBT workflow" />
+          <ToggleSwitch IsChecked="{Binding PreferPsbtWorkflow}" />
+        </DockPanel>
 
-      <DockPanel IsVisible="{Binding !IsWatchOnly}">
-        <TextBlock Text="Auto CoinJoin" />
-        <ToggleSwitch IsChecked="{Binding AutoCoinJoin}" />
-      </DockPanel>
+        <DockPanel IsVisible="{Binding !IsWatchOnly}">
+          <TextBlock Text="Auto CoinJoin" />
+          <ToggleSwitch IsChecked="{Binding AutoCoinJoin}" />
+        </DockPanel>
+      </StackPanel>
+      <StackPanel Spacing="20">
```

Comment:
Shouldn't be visible for WatchOnly wallets.

Reply:
done.